docs/user/azure/install.md

```diff
@@ -0,0 +1,64 @@
+# Cluster Installation
+
+At this point, you are ready to perform the OpenShift installation. See below for an example of an
+IPI install.
+
+## Example: Installer-Provided Infrastructure (IPI)
+
+The steps for performing an IPI-based install are outlined [here][cloud-install]. Following this guide you may begin at
+Step 3: Download the Installer.
+
+### Create Configuration
+
+```console
+[~]$ openshift-install-linux-amd64 create install-config
```

Comment:
This is an artifact of the earlier GitHub release structure.  With the current mirror content, you should probably use `./openshift-install ...` or (with an implicit `PATH` installation) `openshift-install ...`.

Reply:
Fixing the AWS instance of this in #1782.